Affiliated-Project-Policy.md

```diff
@@ -127,11 +127,11 @@ The status of Affiliated and Candidate OBF Projects that are not voted on stays
 
   Affiliated OBF projects that have not been actively maintained for over twelve months may have their affiliation status archived as a result of a review by the OBF Board of Directors. The OBF Board of Directors will conduct such reviews in regular intervals, and will make reasonable efforts to contact the project maintainers before taking action.
 
-## Projects with grandfathered affiliation status
+## Projects with 'Legacied' Affiliation Status
 
-The projects considered OBF member projects prior to enactment of this policy are grandfathered to Affiliated OBF Project status. These projects are the so-called Bio\* Projects, which consist of [BioPerl], [Biopython], [BioJava], [BioRuby], [BioSQL]; [DAS] (Distributed Annotation System); and [EMBOSS] (European Molecular Biology Open Software Suite).
+The projects considered OBF member projects prior to enactment of this policy are legacied to Affiliated OBF Project status. These projects are the so-called Bio\* Projects, which consist of [BioPerl], [Biopython], [BioJava], [BioRuby], [BioSQL]; [DAS] (Distributed Annotation System); and [EMBOSS] (European Molecular Biology Open Software Suite).
 
-Other projects that existed prior to enactment of this policy can be grandfathered to Affiliated or Candidate OBF Project status as well upon approval by the OBF Board of Directors. It is expected that the Board will do so in public session. For pre-existing projects grandfathered to Candidate OBF Project status the minimum waiting period for seeking Affiliate OBF Project status is waived.
+Other projects that existed prior to enactment of this policy can be legacied to Affiliated or Candidate OBF Project status as well upon approval by the OBF Board of Directors. It is expected that the Board will do so in public session. For pre-existing projects legacied to Candidate OBF Project status the minimum waiting period for seeking Affiliate OBF Project status is waived.
```

Comment:
How about replacing "grandfathered to" with "granted" (in both cases)?

I looked back at #44, but don't immediately see any discussion there of this entire clause. It seems like a board approved short-cut of the process, which may have been sense when the policy was new and there were a couple of potential new affiliations on the horizon, but do we still need it?

Reply:
I am happy to replace with 'granted'. As for the other discussion - I suggest having a separate discussion in a different issue. let's close this issue ASAP.